src/Statement.php

```diff
@@ -76,8 +78,9 @@
      * @param \DateTime|null    $stored
      * @param Context|null      $context
      * @param Attachment[]|null $attachments
+     * @param string            $version
      */
-    public function __construct(StatementId $id = null, Actor $actor, Verb $verb, Object $object, Result $result = null, Actor $authority = null, \DateTime $created = null, \DateTime $stored = null, Context $context = null, array $attachments = null)
+    public function __construct(StatementId $id = null, Actor $actor, Verb $verb, Object $object, Result $result = null, Actor $authority = null, \DateTime $created = null, \DateTime $stored = null, Context $context = null, array $attachments = null, $version = '1.0.3')
```

Comment:
Default should be `1.0.0` instead of `1.0.3`.

From the spec : 

> Statements returned by an LRS MUST retain the version they are accepted with. If they lack a version, the version MUST be set to 1.0.0.

And

> **Note:** The requirement for the "version" property to contain the value 1.0.0, rather than the latest patch version is deliberate since Statements in any version of 1.0.x conform to the 1.0.0 data model. In fact, a single Statement may be included in multiple requests over time, each following a different patch version of the specification. The patch version of the specification being followed can be determined from the "X-Experience-API-Version" header being used in each request.